int8 strategy
float64 goal_xmm
float64 goal_ymm
std_msgs/Header header
